tests/Integration/SDK/SpanBuilderTest.php

```diff
@@ -230,6 +236,62 @@ public function test_set_attribute_after_end(): void
         $span->end();
     }
 
+    public function test_set_attribute_null_string_value(): void
+    {
+        /** @var Span $span */
+        $span = $this
+            ->tracer
+            ->spanBuilder(self::SPAN_NAME)
+            ->setAttribute('nil', null)
+            ->setAttribute('empty-string', '')
```

Comment:
We were discussing whether or not an empty string should be an allowed attribute.  Not necessarily part of this PR, just something to raise.